lib/matplotlib/cbook/__init__.py

```diff
@@ -469,7 +469,7 @@ def is_scalar_or_string(val):
     return isinstance(val, str) or not np.iterable(val)
 
 
-def get_sample_data(fname, asfileobj=True):
+def get_sample_data(fname, asfileobj=True, *, np_load=False):
```

Comment:
`np_load` should be mentioned in the docstring.

Reply:
fixed